CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## 1.0.1-dev
```

Comment:
Drop the `-dev` so we can publish ASAP? (same in pubspec.yaml)

Reply:
👍  done!